app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -1326,6 +1326,158 @@ def test_serializer_feature_config_validation_bad_json_value(self):
             feature_values_errors[0]["value"][0],
         )
 
+    @parameterized.expand(
+        [
+            (
+                NimbusExperiment.Application.DESKTOP,
+                NimbusExperiment.Version.FIREFOX_105,
+                False,
+                False,
+            ),
+            (
+                NimbusExperiment.Application.DESKTOP,
+                NimbusExperiment.Version.FIREFOX_101,
+                True,
+                True,
+            ),
+            (
+                NimbusExperiment.Application.DESKTOP,
+                NimbusExperiment.Version.FIREFOX_101,
+                False,
+                True,
+            ),
+            (
+                NimbusExperiment.Application.DESKTOP,
+                NimbusExperiment.Version.FIREFOX_105,
+                True,
+                True,
+            ),
```

Comment:
Yeah 101 and 105 are on either side of the supported version, but it might be clearer if they're right on the line, ie 103 and 104.